### PR TITLE
add RETURNN behavior_version

### PR DESCRIPTION
**Pull request by vieting**
For #92, we need RETURNN's `behavior_version` to have the correct behavior of the `DotLayer`. There might be fixes necessary due to this change.

### CODE REVIEW

File: pytorch_to_returnn/converter/converter.py [hunk 1]
```diff
@@ -294,6 +294,7 @@ def _run_returnn_standalone_net_dict(self):
       config = Config({
         "extern_data": {"data": self._returnn_in_data_dict},
         "debug_print_layer_output_template": True,
+        "behavior_version": 12,
```
Comment (albertz) [inline pytorch_to_returnn/converter/converter.py:297]: I would not hardcode this number, but rather put it somewhere as a global, e.g. to `naming/returnn_ctx.py`.

**Comment by albertz** [comment]
One fix:
Change
```
net_dict={name: {"class": "subnetwork", "from": "data", "subnetwork": {"output": {"class": "copy"}}}}
```
to
```
net_dict={name: {"class": "subnetwork", "from": "data", "subnetwork": {"output": {"class": "copy", "from": "data"}}}}
```

**Comment by albertz** [comment]
Ah we have to change `get_returnn_axis_description`. When you remove the `if dim_tag.dyn_size is not None:` and just always use this code, does this already work?

File: pytorch_to_returnn/torch/nn/modules/batchnorm.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {
       "class": "batch_norm", "from": self._get_input_layer_name(input),
       "update_sample_only_in_training": True, "delay_sample_update": True,
-      "param_version": 1,
+      "param_version": 1, "masked_time": True,
```
Comment (albertz) [inline pytorch_to_returnn/torch/nn/modules/batchnorm.py:99]: This is actually not the same as PyTorch. In PyTorch, the time is not not masked.

Comment (albertz) [reply pytorch_to_returnn/torch/nn/modules/batchnorm.py:99]: You might also want to update the param version. This allows to use the efficient fused batch norm op then.

Comment (vieting) [reply pytorch_to_returnn/torch/nn/modules/batchnorm.py:99]: Right. For the param version, which do you suggest?

Comment (albertz) [reply pytorch_to_returnn/torch/nn/modules/batchnorm.py:99]: param_version = 2. This will change the param name (it gets the prefix "v2_") and it will not add the broadcast dims to the variable.

**Comment by albertz** [comment]
You also need to adapt `BatchNorm1d.import_params_torch_to_returnn` for the new param version.

**Comment by vieting** [comment]
> out_shape = [layer.output.dim]

That seems to work. So we don't neet `_expand_dims` at all anymore, I guess.

**Review by albertz — APPROVED** [review]
Looks all good.
